schema.graphql

```diff
@@ -34,6 +34,7 @@ type SRR @entity {
   id: ID! # tokenId.toString()
   tokenId: String # ERC721 token id
   isPrimaryIssuer: Boolean
+  artist: LicensedUserWallet
   artistAddress: Bytes
```

Comment:
keeping this just for backward compatibility I guess?  otherwise it's redundant.  can be marked deprecated and removed in a later release i guess.